support-cycle.md

```diff
@@ -0,0 +1,180 @@
+## Supported Cycle
+
+To ensure a consistent level of compatibility & support in our projects, each project under the Chai organisations must strictly adhere to a standardised lifecycle support policy, and platform support policy, as document below.
```

Comment:
Nits:

- I think "organizations" should be singular here
- "documented below"